quickwit/quickwit-indexing/src/actors/indexer.rs

```diff
@@ -225,14 +231,11 @@ impl IndexerState {
 /// A workbench hosts the set of `IndexedSplit` that will are being built.
 struct IndexingWorkbench {
     workbench_id: Ulid,
+    span: Span,
+    _indexing_span: Span,
     indexed_splits: FnvHashMap<u64, IndexedSplitBuilder>,
     checkpoint_delta: IndexCheckpointDelta,
     publish_lock: PublishLock,
-    // TODO create this Instant on the source side to be more accurate.
-    // Right now this instant is used to compute time-to-search, but this
-    // does not include the amount of time a document could have been
-    // staying in the indexer queue or in the push api queue.
-    date_of_birth: Instant,
```

Comment:
We don't need `date_of_birth` since the top span of a indexing batch does that job.